qa-jcoin.planx-pla.net/portal/gitops.json

```diff
@@ -708,12 +626,5 @@
     }
   ],
   "showArboristAuthzOnProfile": true,
-  "showFenceAuthzOnProfile": false,
-  "componentToResourceMapping": {
-    "HUB Documentation": {
```

Comment:
are you removing this on purpose?

Reply:
I did not - thanks for catching - I added this back in

Reply:
I did mean to remove the cmc_data_type to make it consistent with PROD